### PR TITLE
Add a url analyzer endpoint

### PR DESCRIPTION
**Pull request by jeroenpf**
This PR introduces a new endpoint that can be called in a similar manner as to how the mshots endpoint can be called.

The goal is to create an endpoint `/url-analyzer?url=www.example.com` that will return information, such as colors, typography, meta, from a given URL. 

Related (child) PRs:
- #54 
- #55 
- #56 
- #59

### CODE REVIEW

**Comment by jeroenpf** [comment]
Please note that this PR will be the **base** of a series of PR's related to this development.  This PR will remain in draft until all related PRs have been merged and this endpoint is considered production ready.

**Review by vishnugopal — APPROVED** [review]
Have tested this again and it LGTM.

**Pull request merged**
